clients/src/test/java/org/apache/kafka/common/serialization/SerializationTest.java

```diff
@@ -84,6 +84,28 @@ public void testStringSerializer() {
     }
 
     @Test
+    public void testShortSerializer() {
+        Short[] shorts = new Short[]{
+            32767,
+            -32768
+        };
+
+        Serializer<Short> serializer = Serdes.Short().serializer();
+        Deserializer<Short> deserializer = Serdes.Short().deserializer();
+
+        for (Short value : shorts) {
+            assertEquals("Should get the original short after serialization and deserialization",
+                    value, deserializer.deserialize(topic, serializer.serialize(topic, value)));
+        }
+
+        assertEquals("Should support null in serialization and deserialization",
+                null, deserializer.deserialize(topic, serializer.serialize(topic, null)));
+
+        serializer.close();
```

Comment:
The close methods are no-ops so there isn't any point in calling them. If they weren't no-ops then this should be surrounded with `try{..}finally{...}` as close won't be called if the test fails.

Yes - i can see that is how it has been done in other tests in this class, but they are also wrong! :-)